blockchain/difficulty.go

```diff
@@ -224,6 +225,26 @@ func (b *BlockChain) calcNextRequiredDifficulty(lastNode *blockNode, newBlockTim
 		return b.chainParams.PowLimitBits, nil
 	}
 
+	config := b.chainParams.LWMA
+	if lastNode.height >= config.EnableHeight {
```

Comment:
Did you consider the difficulty reduce window before the actual mainnet LWMA launch?
https://github.com/BTCGPU/BTCGPU/blob/76f46c4048718a49b52d724720e8620b5daac68f/src/pow.cpp#L52

Reply:
We can skip that check. We can use use checkpoint right after the fork to verify the chains integrity up until that point and continue with 'normal' sync after. (or use an other checkpoint to same some sync time)

Reply:
Can users disable the checkpoint to performa full verification on btgd? If not, it's reasonable to ignore.

Reply:
If they removed the code they can skip it, not sure if there is any other way - maybe there is.

Reply:
Please double check if there's any config or command line flag to disable it. Of course changing of the code is not considered a feature.

Reply:
It seems checkpointing cannot be bypassed 
```// Ensure chain matches up to predetermined checkpoints.
	blockHash := header.BlockHash()
	if !b.verifyCheckpoint(blockHeight, &blockHash) {
		str := fmt.Sprintf("block at height %d does not match "+
			"checkpoint hash", blockHeight)
		return ruleError(ErrBadCheckpoint, str)
	}```